resources/application-connector/charts/connector-service/README.md

```diff
@@ -1,18 +1,29 @@
 # Connector Service
 
 ## Overview
+
 The Connector Service is responsible for generating and sending back client certificates based on Certificate Signing Request (CSR).
 
 ## Configuration
+
 The Connector Service has the following parameters, that can be set through the chart:
 - **appName** - This is the name of the application used by Kubernetes deployments and services. The default value is `connector-service`.
 - **externalAPIPort** - This port exposes the Connector Service API to an external solution. The default port is `8081`.
 - **internalAPIPort** - This port exposes the Connector Service within Kubernetes cluster. The default port is `8080`.
 - **namespace** - Namespace where Connector Service is deployed. The default Namespace is `kyma-integration`.
 - **tokenLength** - Length of registration tokens. The default value is `64`.
-- **tokenExpirationMinutes** - Time after which tokens expire and are no longer valid. The default value is `5` minutes.
-- **domainName** - Domain name of the cluster, used for generating URL. Default domain name is `.wormhole.cluster.kyma.cx`.
-- **certificateServiceHost** - Host at which this service is accessible, used for generating URL. Default host is `cert-service.wormhole.cluster.kyma.cx`.
+- **appTokenExpirationMinutes** - Time after which tokens for applications expire and are no longer valid. The default value is `5` minutes.
+- **runtimeTokenExpirationMinutes** - Time after which tokens for runtimes expire and are no longer valid. The default value is `10` minutes.
+- **caSecretName** - Name of the secret which contains the root Certificate Authority (CA). The default value is `nginx-auth-ca`.
```

Comment:
If I understand it correctly and this `README.md` file provides information about chart parameters, and `components/connector-service/README.md` file provides information about arguments from `options.go` file then this default value should be `connector-service-app-ca`